### ISSUE TITLE
[BUG] blob not loadable from cache if umlaut is in path

### ISSUE THREAD

**Issue by goeck** [luxonis/depthai#684]
Firing up the demo application via Windows CLI after installing all requirements as given in README (via command "python depthai_demo.py") results in GUI being opened but finally in GUI error message, stating that the blob file can't be found.
I assured the blob file is actually existing at the given path. 

![grafik](https://user-images.githubusercontent.com/392197/163421369-4e631dd3-07d8-4420-a745-61651bdc07a6.png)

1. Login as user having an Windows allowed german Umlaut (ä,ü,ö)
2. Run demo script "python depthai_demo.py" 
3. See error

Is it possible to configure the cache path, so I can move it outside the user directory?

**Comment by themarpe** [comment]
Hi @goeck 

Sorry for the issues here - we've just merged in support for UTF-8 paths on Windows as well into the core library.
After we make a release, this issue will be fixed.

Regarding moving the cache directory, CC: @VanDavv

**Comment by VanDavv** [comment]
Yes, it's possible to change the cache dir. After blobconverter import, please add

```
blobconverter.set_defaults(output_dir=<path>)
```

This will make the blobconverter use a specific cache directory instead of the default

**Comment by goeck** [comment]
@VanDavv 
Thanks for the hint, maybe you can guide me just a little bit more, in which file to add this in order to make the demo working? That would be awesome. :-)

**Comment by VanDavv** [comment]
@goeck please add after this line - https://github.com/luxonis/depthai/blob/a8d162cc72780c44c2718ed0b8c718a33e50def0/depthai_sdk/src/depthai_sdk/managers/blob_manager.py#L3

**Comment by goeck** [comment]
Sweet. Thanks a mile!
